tasks/configure.yml

```diff
@@ -14,3 +14,24 @@
       line: "sonar.jdbc.url=jdbc:mysql://{{ sonar_mysql_host }}:{{ sonar_mysql_port }}/{{ sonar_mysql_database }}?useUnicode=true&characterEncoding=utf8&rewriteBatchedStatements=true&useConfigs=maxPerformance"
     - regexp: "^sonar.web.context"
       line: "sonar.web.context={{ sonar_web_context }}"
+    - regexp: "^sonar.ce.javaOpts"
+      line: "sonar.ce.javaOpts={{ sonar_compute_engine_memory_configs }}"
+
+- name: Configure MySQL memory for handling large packet sizes.
+  replace:
+    dest: "{{ mysql_config_file }}"
+    regexp: "{{ item.regexp }}(.+)$"
+    replace: "{{ item.line }}"
+  with_items:
+    - regexp: "^innodb_buffer_pool_size"
+      line: "innodb_buffer_pool_size = {{ mysql_innodb_buffer_pool_size }}"
+    - regexp: "^max_allowed_packet"
+      line: "max_allowed_packet = {{ mysql_max_allowed_packet }}"
+
+- name: Install SonarQube Dependency Check plugin.
+  get_url:
+    url: https://github.com/SonarSecurityCommunity/dependency-check-sonar-plugin/releases/download/1.2.4/sonar-dependency-check-plugin-1.2.4.jar
```

Comment:
Would you consider a list of plugin urls to install so it can be customized by whatever runs the role? Makes it easier to add/upgrade plugins by just updating `site.yml` in the ansible repo.

```
- name: Install SonarQube plugins
  get_url:
    url: "{{ item }}"
    dest: "{{ sonar_directory }}/extensions/plugins"
    owner: sonar
    group: sonar
  with_items: "{{ sonar_plugin_urls }}"
```

in defaults/main.yml
```
sonar_plugin_urls: []
```

In site.yml
```
sonar_plugin_urls: 
  - 'https://github.com/SonarSecurityCommunity/dependency-check-sonar-plugin/releases/download/1.2.4/sonar-dependency-check-plugin-1.2.4.jar'
```

Reply:
good suggestion

Reply:
done, much obliged